.gitattributes

```diff
@@ -0,0 +1,2 @@
+# reclassify ipynb notebooks as Python
+*.ipynb linguist-language=Python
```

Comment:
are there any docs that tell us how this works?

Reply:
https://github.com/github-linguist/linguist/blob/master/docs/overrides.md